tools/telemetry/README.md

```diff
@@ -0,0 +1,22 @@
+### Telemetry tools
+
+Running the services specified in the `docker-compose.yml` file will give you access to:
+
+- The Jaeger UI with both the `/monitor` (latency, errors, req/sec) and `/search` (traces) tabs ready to use.
+- A Prometheus server used by Jaeger with enhanced monitoring data provided by OpenTelemetry.
+
+To get started:
+
+1. Start the necessary services by running `docker compose up` in this directory.
+2. Start the Fleet server with telemetry enabled and configured with this:
+
+```
+OTEL_SERVICE_NAME="fleet" \
+OTEL_EXPORTER_OTLP_ENDPOINT="http://host.docker.internal:4317" \
```

Comment:
If you are running fleet locally where docker is runnig, you should be able to reach `otel-collector` on http://localhost:4317